ownCloud/Resources/Info.plist

```diff
@@ -29,8 +33,14 @@
 		<key>NSAllowsArbitraryLoads</key>
 		<true/>
 	</dict>
+	<key>NSAppleMusicUsageDescription</key>
+	<string>We need this so that you can upload photos and videos from your photo library</string>
```

Comment:
Alternative suggestion:
This permission is needed by the ownCloud app to upload media files to your server.

Reply:
Done.